tools/x-docs/src/docs/guides/components/cookbook/composing-components.md

```diff
@@ -0,0 +1,29 @@
+# Composing components
+
+Some components have slots that allow consumers to add extra elements in certain places. These can be other components or strings of HTML. One example of this is the [Teaser component's "actions" slot](https://github.com/financial-times/x-dash/tree/master/components/x-teaser/src/CustomSlot.jsx), which is typically used for a topic follow button.
```

Comment:
It's just called `customSlot` now

Reply:
Which still makes me gag a bit

Reply:
might need a hand on the wording for this